less/icon-date.less

```diff
@@ -0,0 +1,46 @@
+.icon-date {
```

Comment:
Indentation is incorrect in this file: please use 2-space indentation as in the other LESS files.